src/expr/expr_node.rs

```diff
@@ -8,6 +8,9 @@ use crate::read::*;
 
 const UNKNOWN_QUAL: u8 = b'I';
 
+// Default DNA
+pub const NUC_MAP: [u8; 4] = [b'A', b'C', b'T', b'G'];
```

Comment:
Arrays should be `static` instead of `const` so its not duplicated everywhere by the compiler.